pylint/reporters/ureports/__init__.py

```diff
@@ -75,7 +75,7 @@ def get_table_content(self, table):
             cols -= 1
             result[-1].append(cell)
         # fill missing cells
-        while len(result[-1]) < cols:
+        for _ in range(cols - len(result[-1])):
             result[-1].append("")
```

Comment:
```suggestion
        result[-1].append("" * (cols - len(result[-1])))
```

Reply:
Should be `result[-1] += "" * (cols - len(result[-1])`, right?

Reply:
Hmm, nice catch, it's actually ``result[-1] += [""] * (cols - len(result[-1]))``